includes/admin/class-wc-payments-admin.php

```diff
@@ -360,6 +360,7 @@ public function register_payments_scripts() {
 			'zeroDecimalCurrencies'   => WC_Payments_Utils::zero_decimal_currencies(),
 			'fraudServices'           => $this->account->get_fraud_services_config(),
 			'isJetpackConnected'      => $this->payments_api_client->is_server_connected(),
+			'isJetpackIdcActive'      => \Automattic\Jetpack\Identity_Crisis::has_identity_crisis(),
```

Comment:
This comment is purely optional and opinionated.  You can replace qualifier with an import. I quickly checked the whole class and all qualifiers are replaced with imports. Please have no hesitation to close this comment if you are not agree with me.

Reply:
👍 Addressed 2892e192a14d491dd5aed115f73b6d6600889d74